packages/create-neutrino-project/commands/init/generator/questions.js

```diff
@@ -0,0 +1,140 @@
+const path = require('path');
+
+module.exports = (defaults) => [
+  {
+    name: 'project',
+    type: 'list',
+    message: 'Do you want to build an application, a library, or components?',
+    choices: [
+      { name: 'Application',
+        value: 'application',
+        checked: true
+      },
+      { name: 'Library',
+        value: 'library',
+        checked: false
+      },
+      { name: 'Components',
+        value: 'components',
+        checked: false
+      },
+    ],
+    store: true
+  },
+  {
+    name: 'projectType',
+    type: 'list',
+    message: 'What kind of application?',
+    when: (data) => data.project === 'application',
+    choices: [
+      { name: 'React',
+        value: 'react',
+        checked: true
+      },
+      {
+        name: 'Web',
+        value: 'web',
+        checked: false
+      },
+      { name: 'Node',
+        value: 'node',
+        checked: false
+      }
+    ]
+  },
+  {
+    name: 'projectType',
+    type: 'list',
+    message: 'What kind of library?',
+    when: (data) => data.project === 'library',
+    choices: [{
+      name: 'Web',
+      value: 'taskcluster-web-library',
+      checked: true
+    }]
+  },
+  {
+    name: 'projectType',
+    type: 'list',
+    message: 'What kind of components?',
+    when: (data) => data.project === 'components',
+    choices: [
+      {
+        name: 'React Component',
+        value: 'react-components',
+        checked: false
+      },
+    ]
+  },
+  {
+    name: 'router',
+    type: 'confirm',
+    message: 'Do you want to build a single-page application? I can set up code-splitting for you.',
```

Comment:
Let's rephrase this to be route-based:

"Would you like to use React Router? If so, I will automatically code-split your routes for you."